doc/Changelog.md

```diff
@@ -3,7 +3,21 @@ Changelog {#Changelog}
 
 ## HTTP Image Streaming Service
 
+### 0.3.0
```

Comment:
Yes, I need another PR for changing the version. The build mechanism forces a dedicated PR for changing the version number.